src/app/drive/components/DriveExplorer/DriveExplorerList/DriveExplorerList.tsx

```diff
@@ -429,7 +429,10 @@ const DriveExplorerList: React.FC<DriveExplorerListProps> = memo((props) => {
                 })
               : contextMenuDriveNotSharedLink({
                   shareLink: openLinkSettings,
-                  openPreview: openPreview,
+                  openPreview: () => {
+                    console.log(props.selectedItems[0]);
+                    openPreview(props.selectedItems[0]);
```

Comment:
you better check for that [0] item, it always starts like this, and few weeks later you end debugging a release on a friday @masterprog-cmd

Reply:
Yep, I just was checking something, I forgot to remove all😅